src/libsystemd/sd-netlink/sd-netlink.c

```diff
@@ -614,21 +614,15 @@ int sd_netlink_call_async(
         return k;
```

Comment:
btw, doesn't matter, but we do not use Signed-off-by in systemd that's a kernel thing

Reply:
Yes, matter of habit. I dropped it.